examples/qml_features/tests/tst_custom_base_class.qml

```diff
@@ -82,4 +82,9 @@ TestCase {
         compare(instantiator.count, 1);
         compare(dataChangedSpy.count, 1);
     }
+
+    function test_roles_qenum() {
+        compare(CustomBaseClass.Id, 0);
+        compare(CustomBaseClass.Value, 1);
```

Comment:
Can you add this code to this test, this ensures that enums work in QML through the abstract list model. And show how you can use the imperative API to read the model from QML :-) 

```
    function test_roles_qenum() {
        compare(CustomBaseClass.Id, 0);
        compare(CustomBaseClass.Value, 1);

        const model = createTemporaryObject(componentCustomBaseClass, null, {});

        model.add();
        compare(model.data(model.index(0, 0), CustomBaseClass.Id), 0);
        compare(model.data(model.index(0, 0), CustomBaseClass.Value), 0);

        model.add();
        compare(model.data(model.index(1, 0), CustomBaseClass.Id), 1);
        compare(model.data(model.index(1, 0), CustomBaseClass.Value), 1.0 / 3.0);
    }
```

Reply:
Done.
Also added another QEnum for the background thread state and tested that here.